app/controllers/pages_controller.rb

```diff
@@ -1,4 +1,6 @@
 class PagesController < ApplicationController
+  skip_before_action :authenticate_user!, only: :home
```

Comment:
add :show (as show page is available to view before authentication)

Reply:
added